plugin/adsk/scripts/AEmayaUsdProxyShapeBaseTemplate.mel

```diff
@@ -40,15 +40,39 @@ global proc AEmayaUsdProxyShapePayloadsNew(string $loadPayloads)
     AEmayaUsdProxyShapePayloadsReplace($loadPayloads);
 }
 
+global proc AEMayaUsdProxyShapeStageLoadAll(string $fullNodeName)
+{
+    mayaUsdStageLoadAll $fullNodeName;
+
+    // Maya 2022 and above will automatically refresh the viewport, but
+    // for earlier versions of Maya we need to do it manually.
+    int $mjv = `about -majorVersion`;
+    if ($mjv < 2022) {
+        refresh;
+    }
+}
+
+global proc AEMayaUsdProxyShapeStageUnloadAll(string $fullNodeName)
+{
+    mayaUsdStageUnloadAll $fullNodeName;
+
+    // Maya 2022 and above will automatically refresh the viewport, but
+    // for earlier versions of Maya we need to do it manually.
+    int $mjv = `about -majorVersion`;
+    if ($mjv < 2022) {
+        refresh;
+    }
+}
+
 global proc AEmayaUsdProxyShapePayloadsReplace(string $loadPayloads)
 {
     // From the input attribute we simply want the node name.
     string $nodeName = plugNode($loadPayloads);
     string $res[] = `ls -l $nodeName`;
     string $fullNodeName = $res[0];
 
-    button -e -c ("mayaUsdStageLoadAll(\"" + $fullNodeName + "\")") payloadsLoadAllBtn;
-    button -e -c ("mayaUsdStageUnloadAll(\"" + $fullNodeName + "\")") payloadsUnloadAllBtn;
+    button -e -c ("evalEcho AEMayaUsdProxyShapeStageLoadAll(\"" + $fullNodeName + "\")") payloadsLoadAllBtn;
+    button -e -c ("evalEcho AEMayaUsdProxyShapeStageUnloadAll(\"" + $fullNodeName + "\")") payloadsUnloadAllBtn;
```

Comment:
Used evalEcho here so that pressing the button **will** echo the command in script editor history.

Reply:
OK, I have no idea about this one, but I will just trust you.